trade-finance-manager-ui/server/controllers/case/amendments/amendCoverEndDate.controller.js

```diff
@@ -1,15 +1,26 @@
 const { format, fromUnixTime, getUnixTime } = require('date-fns');
 const { AMENDMENT_STATUS } = require('@ukef/dtfs2-common');
-const { isTfmFacilityEndDateFeatureFlagEnabled } = require('@ukef/dtfs2-common');
+const { HttpStatusCode } = require('axios');
 const api = require('../../../api');
 const { coverEndDateValidation } = require('./validation/amendCoverEndDateDate.validate');
+const { isFacilityEndDateEnabledForTfmFacility } = require('../../helpers/isFacilityEndDateEnabledForTfmFacility');
+
+const getNextPage = (apiResponseStatus, changeFacilityValue, showFacilityEndDatePage, baseUrl) => {
+  if (apiResponseStatus !== HttpStatusCode.Ok) {
```

Comment:
also should the error not be handled by the main controller and this function just return the next page?

Reply:
Good idea, have moved it out